docs/api/migration-guides/qiskit-runtime-examples.mdx

```diff
@@ -130,204 +124,153 @@ op = SparsePauliOp.from_list(
-from qiskit.opflow import StateFn, PauliExpectation, CircuitSampler
-from qiskit import IBMQ
+# Define a real backend
+# backend = provider.get_backend("ibm_brisbane")
 
-# Define the state to sample
-measurable_expression = StateFn(opflow_op, is_measurement=True).compose(opflow_state)
+# Circuits must obey the Instruction Set Architecture (ISA) of a particular backend.
+from qiskit.transpiler.preset_passmanagers import generate_preset_pass_manager
 
-# Convert to expectation value calculation object
-expectation = PauliExpectation().convert(measurable_expression)
+pm = generate_preset_pass_manager(backend=backend, optimization_level=1)
+isa_state = pm.run(state)
+```
+  </TabItem>
+</Tabs>
 
-# Define provider and backend
-provider = IBMQ.get_provider(hub="ibm-q", group="open", project="main")
-backend = provider.get_backend("ibmq_qasm_simulator")
 
-# Inject backend into circuit sampler
-sampler = CircuitSampler(backend).convert(expectation)
+<span id="legacy-opflow"></span>
+##### 1.a. Legacy only: Convert problem to opflow
 
-# Evaluate
-expectation_value = sampler.eval().real
-```
+`qiskit.opflow` provided classes to represent operators
+and quantum states, so the problem defined above would be wrapped as:
 
 ``` python
->>> print("expectation: ", expectation_value)
-expectation:  -1.065734058826613
+from qiskit.opflow import CircuitStateFn, PauliSumOp
+
+opflow_op = PauliSumOp(op)
+opflow_state = CircuitStateFn(isa_state)
 ```
 
-##### 2.b. Updated: Use the Estimator Runtime primitive
+This step is not necessary when using the primitives. For instructions to migrate from `qiskit.opflow`, see the [Opflow migration guide](./qiskit-opflow-module).
 
-The `Estimator` simplifies the user-side syntax, making it a more
-convenient tool for algorithm design.
+#### 2. Calculate expectation values
 
-<Admonition type="note">
-    Replace `ibmq_qasm_simulator` with your device name to see the complete workflow for a real device.
-</Admonition>
+<Tabs>
+  <TabItem value="updated" label="Runtime Estimator">
+Estimator simplifies the user-side syntax, making it a more
+convenient tool for algorithm design.
 
 ``` python
-from qiskit_ibm_runtime import QiskitRuntimeService, Estimator
-
-service = QiskitRuntimeService()
-backend = service.backend("ibmq_qasm_simulator")
+from qiskit_ibm_runtime import EstimatorV2 as Estimator
 
-estimator = Estimator(session=backend)
+estimator = Estimator(backend, options={"default_shots": int(1e4)})
+job = estimator.run([(isa_state, isa_observables)])
 
-expectation_value = estimator.run(state, op).result().values
+# Get results for the first (and only) PUB
+pub_result = job.result()[0]
 ```
 
 Note that the Estimator returns a list of values, as it can perform batched evaluations.
 
 ``` python
->>> print("expectation: ", expectation_value)
-expectation:  [-1.06329149]
+>>> print(f">>> Expectation values: {pub_result.data.evs}")
+>>> Expectation values: [-1.05237325 -0.20984981  0.37345495 -0.00564005 -0.00141352]

```

Comment:
I didn't get to re-run the example so I don't know if the output is between square brackets or not:

```suggestion
>>> print(f">>> Expectation value: {pub_result.data.evs}")
>>> Expectation value: [-1.06329149]

```